crates/editor/src/test/editor_test_context.rs

```diff
@@ -42,17 +43,21 @@ impl EditorTestContext {
     pub async fn new(cx: &mut gpui::TestAppContext) -> EditorTestContext {
         let fs = FakeFs::new(cx.executor());
         // fs.insert_file("/file", "".to_owned()).await;
+        #[cfg(windows)]
```

Comment:
We have this already: https://github.com/zed-industries/zed/pull/19773/files#diff-5746e81b1a1164aba8f63ef0831eed5647c74317671f9c781b998a7994e57045R318

Let's either create the very same method in this struct, or, better, find a common struct/enum/module where we can extract this away.